jsquiz.js

```diff
@@ -0,0 +1,77 @@
+//create key obj with two key value pairs
+var tree = {
+  height: "",
+  sym: ""
+};
+
+//create variable to store input elements
+var inputHeight = document.getElementById('height');
+var inputSymbol = document.getElementById('symbol');
+
+//create variable to store button element
+var grow = document.getElementById('grow');
+
+
+//create event listener for click
+grow.addEventListener('click', handleGrowTree);
+
+//create event listener for enter key on input elements
+inputHeight.addEventListener('keypress', handleEnterKeyPress);
+inputSymbol.addEventListener('keypress', handleEnterKeyPress);
+
+
+//define function for enter key press
+function handleEnterKeyPress (event) {
+
+  //determine if enter key is pressed
+  if ( event.keyCode === 13 ) {
+    //determine if each fields have a value, if not alert
+    if ( inputHeight.value === "" | inputSymbol.value === "" ) {
+      alert('Error: Both input fields need a value');
+
+      //if values are entered invoke grow tree function
+    } else {
+      handleGrowTree();
+    };
+  }; //end of first condition for enter key
+};
+
+//define function to print tree to console
+function handleGrowTree() {
```

Comment:
Review the requirements. This function needs to take an object as an argument. When you call `handleGrowTree` you need to pass the `tree` object into it.